emu/src/cpu/arm/instructions.rs

```diff
@@ -77,8 +77,7 @@ pub enum ArmModeInstruction {
         destination: u32,
         op2: AluSecondOperandInfo,
     },
-    Multiply,
-    MultiplyLong,
+    Multiply(ArmModeMultiplyKind),
```

Comment:
I'm not 100% sure about this, just because semantically the `ArmModeInstruction` maps every possible opcode as per armv7tdmi datasheet.

Do you think we could keep the old `Multiply` and `MultiplyLong` or are there any specific reason (maybe it's easier to implement like this?) to keep things under `Multiply` only?


cc @guerinoni

Reply:
Also, I put tests for muls in `emu/src/cpu/arm7tdmi.rs` perhaps I should move them to `emu/src/cpu/arm/operations.rs`, right?